file/provider/create_volume.go

```diff
@@ -63,6 +63,21 @@ func (vpcs *VPCSession) CreateVolume(volumeRequest provider.Volume) (volumeRespo
 		},
 	}
 
+	// Check for VPC ID, SubnetID or PrimaryIPID either of the one is mandatory for VolumeAccessPoint/FileShareTarget creation
```

Comment:
Could you please add more documentation for this condition.

Reply:
Done

Reply:
Than you @sameshai